VirtoCommerce.Storefront/Startup.cs

```diff
@@ -373,6 +374,22 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddResponseCompression();
 
+            // Image Processor - correct image URL with image processing service endpoint
+            var imageProcessorOptions = new ImageProcessorOptions();
+            Configuration.GetSection("VirtoCommerce:ImageProcessor").Bind(imageProcessorOptions);
+            if (imageProcessorOptions.Provider.EqualsInvariant("Imagekit"))
+            {
+                services.AddSingleton<IImageProcessor, ImagekitImageProcessor>();
+                services.Configure<ImageProcessorOptions>(options =>
+                {
+                    options.ServiceUrl = imageProcessorOptions.ServiceUrl;
+                });
+            }
```

Comment:
you must add options like this:
services.Configure<ImageProcessorOptions>(Configuration.GetSection("VirtoCommerce:ImageProcessor"))
and lines 383-386 are unnecessarily

Reply:
yes, you are right